callback_plugins/human_log.py

```diff
@@ -18,37 +19,55 @@
     }
 
 class CallbackModule(default_CallbackModule):
-    def __display_result(self, result, color, ignore_errors=None):
+    def __display_result(self, result, color, ignore_errors=None, loop_idx=0):
         if ignore_errors is not None:
             self._display.display(f"==> FAILED | ignore_errors={ignore_errors}", color=color)
 
 
         try: self._display.display(f"msg: {result._result['msg']}", color=color)
         except KeyError: pass
 
-        try:
-            cmd = result._result['cmd']
+        if "results" in result._result:
```

Comment:
for things like `result._result` is that I don't get python ;P

Reply:
`_result` means that this field is internal to the class of `result`, it's only a naming convention (`__result` is stronger, and the languages helps hiding it by mangling it into something like `_classname_result`)